WordPress/src/main/java/org/wordpress/android/ui/accounts/SignInFragment.java

```diff
@@ -859,16 +859,19 @@ private void refreshBlogContent(Map<String, Object> blogMap) {
     }
 
     /**
-     * Get first blog and call RefreshBlogContentTask. First blog will be autoselected when user login.
+     * Get the first ten blogs and call RefreshBlogContentTask. First blog will be autoselected when user login.
      * Also when a user add a new self hosted blog, userBlogList contains only one element.
      * We don't want to refresh the whole list because it can be huge and each blog is refreshed when
      * user selects it.
      */
-    private void refreshFirstBlogContent() {
+    private void refreshFirstTenBlogsContent() {
         List<Map<String, Object>> visibleBlogs = WordPress.wpDB.getBlogsBy("isHidden = 0", null, 1);
```

Comment:
Right, not sure why isn't there the correct value. I probably did an unexpected undo.